src/set_theory/game/pgame.lean

```diff
@@ -1396,4 +1423,44 @@ begin
     { exact ⟨sum.inl punit.star, (zero_add_equiv _).2⟩ } }
 end
 
+/-- The pre-game `up` is defined as `{0 | *}`. It's positive but smaller than all positive numeric
+games. -/
```

Comment:
Is this a name you invented or one from the literature?

Reply:
It's in the literature. Up is mentioned a few times in ONAG.